### DISCUSSION TITLE
窗口大小被限制了

### DISCUSSION THREAD

**Discussion by yybbwc** [hooke007/mpv_PlayKit#119] [mpv 通用]
我的系统版本是win8.1。

最近发现，如果视频以以原尺寸进行播放，就会比较清晰，而如果进行了缩放就会比较模糊。所以我倾向于使用原尺寸来播放视频。

但是mpv的最大窗口大小有限制，不能超过屏幕的大小。即使我尝试使用autofit参数来扩宽它的窗口大小限制，mpv也只是显示参数无效。

这一个限制导致我的某些视频被缩放到了0.955、0.977等窗口大小，视频变得模糊起来

有没有办法让mpv的窗口大小超过屏幕的大小？

**Comment by hooke007** [comment]
这是win的限制 https://github.com/mpv-player/mpv/issues/9723#issuecomment-1031262997

> 进行了缩放就会比较模糊

用锐利的算法进行缩放
